http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ZombieRuntimeMonitor.scala

```diff
@@ -48,37 +54,69 @@ class ZombieRuntimeMonitor[F[_]: Parallel: ContextShift: Timer](
+      // Get active runtimes from the Leo DB, grouped by project
+      activeRuntimeMap <- ZombieMonitorQueries.listActiveZombieQuery.transaction
 
       _ <- logger.info(
-        s"Starting zombie detection across ${clusterMap.size} projects with concurrency of ${config.concurrency}"
+        s"Starting active zombie detection across ${activeRuntimeMap.size} projects with concurrency of ${config.concurrency}"
       )
-      zombies <- clusterMap.toList.parFlatTraverse[F, PotentialZombieRuntime] {
-        case (project, clusters) =>
+      activeZombies <- activeRuntimeMap.toList.parFlatTraverse[F, ZombieCandidate] {
+        case (project, zombieCandidates) =>
           semaphore.withPermit(
             // Check if the project is active
             isProjectActiveInGoogle(project).flatMap {
               case true =>
-                // If the project is active, check each individual cluster
-                clusters.toList.traverseFilter { cluster =>
-                  isRuntimeActiveInGoogle(cluster, startInstant).ifA(F.pure(None), F.pure(Some(cluster)))
+                // If the project is active, check each individual runtime
+                zombieCandidates.toList.traverseFilter { candidate =>
+                  val runtime = candidate.runtime
+                  // If the cluster is less than one minute old, it may not exist in Google because it is still Provisioning, so it's not a zombie
+                  isRuntimeActiveInGoogle(runtime.googleProject, runtime.runtimeName, candidate.cloudService, startInstant)
+                    .ifA[Option[ZombieCandidate]](F.pure(None), zombieIfOlderThanOneMinute(candidate, startInstant))
                 }
               case false =>
-                // If the project is inactive, all clusters in the project are zombies
-                F.pure(clusters.toList)
+                // If the project is inactive, all runtimes in the project are zombies
+                F.pure(zombieCandidates.toList)
             }
           )
       }
-      // Error out each detected zombie cluster
-      _ <- zombies.parTraverse(zombie => semaphore.withPermit(handleZombieRuntime(zombie, startInstant)))
+
+      // Get all deleted cluster that haven't been confirmed from the Leo DB
+      unconfirmedDeletedRuntimes <- ZombieMonitorQueries.listInactiveZombieQuery.transaction
+
+      _ <- logger.info(
+        s"Starting inactive zombie detection within ${unconfirmedDeletedRuntimes.size} runtimes with concurrency of ${config.concurrency}"
+      )
+
+      inactiveZombies <- unconfirmedDeletedRuntimes.traverseFilter {
+        candidate =>
+          val runtime = candidate.runtime
+          semaphore.withPermit(
+            // If the cluster is less than one minute old, it may not exist in Google because it is still Provisioning, so it's not a zombie
+            isRuntimeActiveInGoogle(runtime.googleProject, runtime.runtimeName, candidate.cloudService, startInstant)
```

Comment:
I'm worried this might lead to a weird state - what if a runtime is `Deleting` on Google? I think trying to delete a `Deleting` runtime would cause an error right?